app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id
-  has_many :likes
-  has_many :Comments
+  has_many :comments, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
```

Comment:
- [optional] your MIT link is broken. please kindly fix that. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
 
![Screenshot (1313)](https://github.com/ahmed025566/Blog_App/assets/65298048/f6d035c0-3724-4f47-ab4e-2177d11add2b)